source/ed/connectors/fusio_parser.cpp

```diff
@@ -436,6 +436,7 @@ void StopTimeFusioHandler::init(Data& data) {
     }
     id_c = csv.get_pos_col("stop_time_id");
     headsign_c = csv.get_pos_col("stop_headsign");
+    short_name_c = csv.get_pos_col("trip_short_name_at_stop");
```

Comment:
```suggestion
    trip_short_name_at_stop_c = csv.get_pos_col("trip_short_name_at_stop");
```
For sake of consistency